dialme/README.md

```diff
@@ -0,0 +1,40 @@
+# Simple protocol to request dial backs
+
+> Allow a peer in a network to be dialed back after an initial connection has been negotiated
+
+In certain scenarios, it might be desirable for a peer to ask another peer to dial it back after an initial connection has been established. This might be useful in scenarios where the dialing peer has learned enough about the network topology and it knows of a better route to be reached over (NATed peers vs non NATed peers). This protocol will allow it to ask its counterpart to dial it back on a hopefully better route.
+
+#### Requirements
+
+- Should be transports agnostic
+- Should be connection agnostic
+- Can work with or without connection upgrades
```

Comment:
couldnt this all be simplified by saying that the dialme protocol is a standard libp2p protocol? all libp2p protocols are transport and connection agnostic and assume nothing about the network stack implementation